test/run_tests.c

```diff
@@ -26,5 +26,6 @@ int main(void)
 	describe("test block layout", test_block_layout);
 	describe("test flex layout", test_flex_layout);
 	describe("test widget rect", test_widget_rect);
+	// describe("test clipboard", test_clipboard);
```

Comment:
```diff
- // describe("test clipboard", test_clipboard);
+ describe("test clipboard", test_clipboard);
```

Reply:
Yes, I'm just trying to figure out here why the test is hanging